app/charges/controllers.py

```diff
@@ -190,7 +190,7 @@ def create_charge(user, user_data):
     charge.author = user.id
     charge.description = user_data.get("description", "")
     charge.committee = committee.id
-    charge.status = 0
+    charge.status = int(user_data.get("status", ""))
```

Comment:
Should this be `user_data.get("status", 0)`?

Reply:
When i wrote this i was following the convention of the other attributes in the file. It seems to work the same with all three of these options:

user_data.get("status", "")
user_data.get("status", 0)
user_data.get("status")

I'm not sure why the second parameter was ever included in the first place. Since it isn't causing problems i will leave it the way it is for now.